Make Failure

## Question
<!--- Ask a question -->

I conducted 'make' but the following error occurred.
I checked the Makefile includes -lm (linking math header)
Do you have an idea?


make[2]: Leaving directory '/home/jonggyu/Research/HotStorage/tools/open-cas-linux/modules'
make build
make[2]: Entering directory '/home/jonggyu/Research/HotStorage/tools/open-cas-linux/casadm'
  LD  casadm
casadm.a(cas_lib.o): In function `partition_get_config':
cas_lib.c:(.text+0x388d): undefined reference to `roundf'
collect2: error: ld returned 1 exit status
Makefile:103: recipe for target 'casadm' failed
make[2]: *** [casadm] Error 1
make[2]: Leaving directory '/home/jonggyu/Research/HotStorage/tools/open-cas-linux/casadm'
Makefile:92: recipe for target 'all' failed
make[1]: *** [all] Error 2
make[1]: Leaving directory '/home/jonggyu/Research/HotStorage/tools/open-cas-linux/casadm'
Makefile:22: recipe for target 'casadm' failed
make: *** [casadm] Error 2

## Your Environment
<!--- Not obligatory, but the answer may depend on this section -->
<!--- Include as many relevant details about the environment you're working in -->
* OpenCAS version (commit hash or tag):2272127b5035e43726a854c4078203abbcfcb140
* Operating System:Ubuntu 18.04 (gcc version is 7.5.0)
* Kernel version:5.4.0-60-generic
* Cache device type (model):
* Core device type (model):
* Cache configuration:
* * Cache mode: (default: wt)
* * Cache line size: (default: 4)
* * Promotion policy: (default: always)
* * Cleaning policy: (default: alru)
* * Sequential cutoff policy: (default: full)
* Other (e.g. _lsblk_, _casadm -P_, _casadm -L_)

--- a/casadm/Makefile
+++ b/casadm/Makefile
@@ -101,7 +101,7 @@ sync:
 #
 $(TARGET): $(TARGET).a
 	@echo "  LD " $@
-	@$(CC) $(CFLAGS) $(LDFLAGS) -o $(TARGET) $<
+	@$(CC) $(CFLAGS) -o $(TARGET) $< $(LDFLAGS)
 
 $(TARGET).a: $(patsubst %,$(OBJDIR)%,$(OBJS))
 	@echo "  AR " $@

